### PR TITLE
Add instructions for face detection

### PR DESCRIPTION
**Pull request by sovrasov**
Instruction contains description of taining/val scripts and related tools.

### CODE REVIEW

**Comment by sovrasov** [comment]
@AlexanderDokuchaev @snosov1

File: pytorch_toolkit/object_detection/README_face_detection.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+2. Create virtual environment `bash init_venv.sh`
+
+## Training
+
+1. Download pre-trained MobileNetV2 weights `mobilenet_v2.pth.tar` from: [https://github.com/tonylins/pytorch-mobilenet-v2](https://github.com/tonylins/pytorch-mobilenet-v2). Move the file with weights to the folder `snapshots`.
+2. Run in terminal: `python3 ../../external/mmdetection/tools/train.py ../../external/mmdetection/configs/wider_face/mobilenetv2_tiny_ssd300_wider_face.py` to train the detector on single GPU.
+
+## Validation
+
+1. Run in terminal `python ../../external/mmdetection/tools/test.py ../../external/mmdetection/configs/wider_face/mobilenetv2_tiny_ssd300_wider_face.py <CHECKPOINT_PATH> --out result.pkl` to dump detections.
```
Comment (AlexanderDokuchaev) [inline pytorch_toolkit/object_detection/README_face_detection.md]: Write `python3` in all commands

File: pytorch_toolkit/object_detection/README_face_detection.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+│   │   ├── val.txt
+│   │   ├── train.txt
+
+```
+2. Create virtual environment `bash init_venv.sh`
+
+## Training
+
+1. Download pre-trained MobileNetV2 weights `mobilenet_v2.pth.tar` from: [https://github.com/tonylins/pytorch-mobilenet-v2](https://github.com/tonylins/pytorch-mobilenet-v2). Move the file with weights to the folder `snapshots`.
+2. Run in terminal: `python3 ../../external/mmdetection/tools/train.py ../../external/mmdetection/configs/wider_face/mobilenetv2_tiny_ssd300_wider_face.py` to train the detector on single GPU.
```
Comment (AlexanderDokuchaev) [inline pytorch_toolkit/object_detection/README_face_detection.md]: Better write as
```
1. Description of step
'''
command arg1 \
   arg2 \
   arg3
'''
```
And use multiline format for long commands

File: pytorch_toolkit/object_detection/README_face_detection.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+## Conversion to OpenVINO format
+
+1. Convert PyTorch model to ONNX format: run script in terminal `python tools/onnx_export.py ../../external/mmdetection/configs/wider_face/mobilenetv2_tiny_ssd300_wider_face.py <CHECKPOINT> face_detector.onnx`. It produces `face_detector.onnx`.
+2. Convert ONNX model to OpenVINO format with Model Optimizer: run in terminal `python <OpenVINO_INSTALL_DIR>/deployment_tools/model_optimizer/mo.py --input_model face_detector.onnx --scale 255 --reverse_input_channels`. This produces model `face_detector.xml` and weights `face_detector.bin` in single-precision floating-point format (FP32). The obtained model expects normalized image in planar BGR format.
+
+
+## Python demo
+
+To run the demo connect a webcam end execute command:
+* `python tools/detection_live_demo.py ../../external/mmdetection/configs/wider_face/mobilenetv2_tiny_ssd300_wider_face.py <CHECKPOINT> --cam_id 0`
```
Comment (AlexanderDokuchaev) [inline pytorch_toolkit/object_detection/README_face_detection.md]: No need to add * before command

File: pytorch_toolkit/object_detection/README_face_detection.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+## Requirements
+
+* Ubuntu 16.04
+* Python 3.5 or 3.6 (3.6 is preferable)
+* PyTorch 1.1
+
+## Prerequisites
+
+1. Launch script to finally prepare your working environment: `bash prepare_mmdet.sh`
```
[the remark this answers is not in the document]
Comment (sovrasov) [reply pytorch_toolkit/object_detection/README_face_detection.md]: Ok, added it
